1-mandatory/1-writers.js

```diff
@@ -62,19 +63,22 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  let writerNalive40 = writers.filter(writer => !writer.alive && 40< writer.age && writer.age<50);
```

Comment:
Look closely at this code 
```javascript
40< writer.age && writer.age<50
```
Will an author who is exactly 40 be included in the condition?

Reply:
do we wanna use `<=` or `<` based on the requirement of 

> Only `console.log()` out the writers who are in their 40s (meaning between 40 and 49)